### ISSUE TITLE
Adding audio stream

### ISSUE THREAD

**Issue by Alfiegerner** [kroo/wyzecam#67]
## 🚀 Feature Request

Is it possible to add audio, either to recv_video_data or maybe add another method?  

## 🔈 Motivation

Potentially lots of reasons e.g. baby monitor, child watching

**Comment by kroo** [comment]
Yeah, it definitely is, though we would either have to access it in a separate thread, or interleaved video and audio...  this is definitely something worth prioritizing.

**Comment by drinfernoo** [comment]
After seeing the (really cool) new Wyze Bridge Docker container, I'd really love to see this get figured out so that he can implement it as well :+1:

**Comment by AdiAbuAli** [comment]
Hope this will be very soon ... maybe as well Audio out!

**Comment by parthmodi** [comment]
Came here to add a request for this as well!
